latrine_django/frontend/README.md

```diff
@@ -0,0 +1,2434 @@
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
this is the longest readme i think i've ever seen :D , do we need it in here? i guess if people find it useful it should stay but it feels so large and generator-y with lots of information that can be found lots of other places.

Reply:
Yeah, it was generated automatically with create-react-app. We can get rid of it.